console/src/components/Services/Data/TableCommon/TableHeader.js

```diff
@@ -99,7 +104,7 @@ const TableHeader = ({
                 }
                 data-test="table-insert-rows"
               >
-                Insert Row
+                {tabName === 'edit' ? 'Edit' : 'Insert'} Row
```

Comment:
Can we just have an extra tab called `Edit` "only" at the `:tableName/edit` route?
Tabs magically switching functionalities is a bit confusing.